### PR TITLE
New parser: pcs_config

### PR DESCRIPTION
**Pull request by PetrLukac**
This parser is required for incoming rules.

### CODE REVIEW

File: insights/parsers/pcs_config.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
+"""
+PCSConfig - command ``pcs config``
+==================================
+
+This module provides class ``PCSConfig`` for parsing output of ``pcs config`` command.
+Typical ``/usr/sbin/pcs config`` output looks something like:
```
Comment (bfahr) [inline insights/parsers/pcs_config.py]: `like:` should be `like::`

There are some other docstring errors below as well.  To see these, follow the instructions here to generate the docs and review the output: https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring

File: insights/parsers/pcs_config.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+
+@parser(Specs.pcs_config)
+class PCSConfig(CommandParser):
+    """Class to process the output of ``pcs config`` command.
+
+    Attributes:
+        data (dict): A dictionary containing all line. Keys sorted based on keywords of the output.
+            the form::
+            {
```
Comment (bfahr) [inline insights/parsers/pcs_config.py]: There should be a blank line after `the form::` and the dictionary example should be indented 1 tab.

File: insights/parsers/pcs_config.py [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+                ],
+                "Cluster Properties": [
+                    "cluster-infrastructure: corosync",
+                    "cluster-name: cluster-1"
+                ]
+            }
+    Methods:
+        get_resources_clones(self) (dict): Returns a dictionary of ``Clones`` in ``Resources`` keyed with their name.
+            the form::
+            {
```
Comment (bfahr) [inline insights/parsers/pcs_config.py]: There should be a blank line after `the form::` and the dictionary example should be indented 1 tab.  Same with the next method.

File: insights/parsers/pcs_config.py [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+                        "cluster-name: cluster-1"
+                    ]
+                }
+
+    Methods:
+        get_resources_clones(self) (dict): Returns a dictionary of ``Clones`` in ``Resources`` keyed with their name.
+            the form::
+
+                {
+                    "clone-1": [
```
[the remark this answers is not in the document]
Comment (xiangce) [reply insights/parsers/pcs_config.py]: Or if you insist to get all the clones by using `get_resources_clones()`, I think the result could be a list like:

```python
[
 {
  "Clone": "clone-1",
  "Meta Attrs": "interleave=true ordered=true",
  "Resources": [ 
     {
       "Resource": "res-1 (class=ocf provider=pacemaker type=controld)":
       "Operations": [
         "start interval=0s timeout=90 (dlm-start-interval-0s)",
         "stop interval=0s timeout=100 (dlm-stop-interval-0s)",
         "monitor interval=30s on-fail=fence (dlm-monitor-interval-30s)"]
    },
     {
  ]
 },
 {
  "Clone": "clone-2",
 },
]
```

**Comment by PetrLukac** [comment]
I've decided to remove methods get_resources_clones and get_resources_groups. For the purpose of this parser, most information can be found just using `get` method. The two methods provided were unnecessary complicated.

**Comment by xiangce** [comment]
looks good to me

**Pull request merged**